### PR TITLE
Prepare gles-user-module

### PR DESCRIPTION
**Pull request by iartemenko**

### CODE REVIEW

File: recipes-domx/meta-xt-prod-vgpu/recipes-graphics/mesa/mesa_%.bbappend [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# We need this header file to build gles-user-module. Since it was removed
+# by mesa recipe we have to install it manually.
+do_install_append () {
```
Comment (andr2000) [inline recipes-domx/meta-xt-prod-vgpu/recipes-graphics/mesa/mesa_%.bbappend]: you can also put a not, that for example, for AGL it is installed as a part of Qt in the same way

File: recipes-domx/meta-xt-prod-vgpu/recipes-graphics/gles-module/gles-module-egl-headers_1.9.bbappend [hunk 2]
```diff
@@ -0,0 +1,13 @@
+do_install() {
+    # Install header files
```
Comment (andr2000) [inline recipes-domx/meta-xt-prod-vgpu/recipes-graphics/gles-module/gles-module-egl-headers_1.9.bbappend]: can we instead modify the original recipe and copy OpenCL files only if they are present?

File: recipes-domx/meta-xt-prod-vgpu/recipes-graphics/mesa/mesa_%.bbappend (hunk 1 above)
Comment (iartemenko) [reply recipes-domx/meta-xt-prod-vgpu/recipes-graphics/mesa/mesa_%.bbappend]: Done

File: recipes-domx/meta-xt-prod-vgpu/recipes-graphics/gles-module/gles-module-egl-headers_1.9.bbappend (hunk 2 above)
Comment (iartemenko) [reply recipes-domx/meta-xt-prod-vgpu/recipes-graphics/gles-module/gles-module-egl-headers_1.9.bbappend]: Done

**Pull request merged**